blueprints/load-scheduling/promql/config.libsonnet

```diff
@@ -10,6 +10,12 @@ serviceProtectionDefaults {
     * @param (policy.setpoint: float64) Setpoint.
     */
     setpoint: '__REQUIRED_FIELD__',
+    /**
+    * @param (policy.service_protection_core.overload_condition: string)Overload condition determines the criteria to determine overload state. The condition must be one of: gt, lt, gte, lte.
+    */
+    service_protection_core+: {
+      overload_condition: '__REQUIRED_FIELD__',
```

Comment:
This should be hardcoded for each recipe

Reply:
This is hard-coded in all of them except promql as query could be anything here

Reply:
I see